rosdep/python.yaml

```diff
@@ -4276,6 +4276,7 @@ python-transforms3d-pip:
   debian:
     pip:
       packages: [transforms3d]
+  nixos: [python3Packages.transforms3d]
```

Comment:
There's no real reason to add to this one; this is implicitly Python 2, which is no longer supported.

```suggestion
```